Dependencies/xr/Source/OpenXR/SceneUnderstanding.cpp

```diff
@@ -0,0 +1,497 @@
+#define NOMINMAX // avoid including macro max definitions in favor of limit max definitions
+
+#include "SceneUnderstanding.h"
+#include "XrRegistry.h"
+
+using namespace xr;
+
+constexpr SceneUnderstanding::DetectionBoundaryType DEFAULT_BOUNDARY_TYPE = SceneUnderstanding::DetectionBoundaryType::Sphere;
+constexpr float DEFAULT_SPHERE_RADIUS = 5.f;
+constexpr double DEFAULT_UPDATE_INTERVAL_IN_SECONDS = 2;
+constexpr double NANOSECONDS_IN_SECOND = 1000000000;
+
+inline xr::Pose XrPoseToBabylonPose(XrPosef pose)

```

Comment:
If this function is only used in this file (I'm assuming it is) add the `static` qualifier to keep it local to this TU